src/ObjectGraph.es6.js

```diff
@@ -387,28 +399,27 @@ ObjectGraph.prototype.capture = function(o, opts) {
 ObjectGraph.prototype.removeRefs_ = function(id, ids) {
   let found = false;
   if ( this.invData[id] ) {
-    _.forOwn(
-      this.invData[id],
-      (refIds, refKey) => refIds.forEach(
-        refId => {
-          found = true;
-          delete this.data[refId][refKey];
-        }
-      )
-    );
+    var refKeys = Object.keys(this.invData[id]);
+    refKeys.forEach(refKey => {
+      var refIds = refKeys[refKey];
+      refIds.forEach(refId => {
+        found = true;
+        delete this.data[refId][refKey];
+      });
+    });
   }
-  if ( this.invProtos[id] ) {
+  var invProtoIds = this.invProtos[id];
+  if ( invProtoIds ) {
     var newProto = id;
     while (!this.isType(newProto) &&
            ids.some(protoId => protoId === newProto))
       newProto = this.getPrototype(newProto);
 
-    _.forOwn(this.invProtos[id],
-             protoId => {
-               found = true;
-               console.assert(this.protos[protoId] === id);
-               this.protos[protoId] = newProto;
-             });
+    invProtoIds.forEach(invProtoId => {
```

Comment:
Was there a data model change here? `_.forOwn` will iterate over properties and `a.forEach` will iterate over array elements, right?